tests/test_restore_names.py

```diff
@@ -1,3 +1,35 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_function_nothing_to_return():
+    users = [
+        {
+            "first_name": None,
+            "last_name": "Holy",
+            "full_name": "Jack Holy",
+        },
+        {
+            "last_name": "Adams",
+            "full_name": "Mike Adams",
+        },
+    ]
+
+    assert restore_names(users) is None
+
+
+def test_is_function_set_correct_first_name():
```

Comment:
```suggestion
def test_function_should_set_correct_first_name():
```
It's better to specify how the function should behave in the name of the test